src/hooks/useWindowSize.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
```

Comment:
This hook looks very generic. Post this work, I would love it if we proposed this for a more generic place for commonly usable react hooks?

Reply:
AGREE. I put it in the most generic place I could for this repo. I'm going to move the whole icon component into paragon, so this hook will also go into paragon, so from there I think it's just a question of documenting it?

Reply:
yup! paragaon may be the correct place perhaps, but let's check with fedx too yeah